gui/gui/src/TGSpeedo.cxx

```diff
@@ -412,7 +412,10 @@ void TGSpeedo::SetScaleValue(Float_t val)
       fPeakVal = fValue;
 
    if (fBufferSize > 0) {
-      fBuffer[fBufferCount % fBufferSize] = fValue;
+      if (fBufferCount < fBufferSize)
```

Comment:
Instead:
if(fBuffer.size() < fBufferCount+1)
// this change is needed because of resetting fBufferCount to zero on every cycle to prevent the int overflows

Reply:
I suggested the change in https://github.com/bellenot/root/pull/1/files
Thanks!